scripts/build-openapi.js

```diff
@@ -3,111 +3,141 @@
 const fs = require("fs")
 const OAS = require("oas-normalize")
 const swaggerInline = require("swagger-inline")
-const { exec } = require("child_process");
+const { exec } = require("child_process")
 
-const isDryRun = process.argv.indexOf('--dry-run') !== -1;
+const isDryRun = process.argv.indexOf("--dry-run") !== -1
 
 // Storefront API
 swaggerInline(
-  ["./packages/medusa/src/models", "./packages/medusa/src/api/middlewares" , "./packages/medusa/src/api/routes/store"],
+  [
+    "./packages/medusa/src/models",
+    "./packages/medusa/src/types",
```

Comment:
Note:
Adding `./packages/medusa/src/types` to `swaggerInline` was the only change in this file.